playbooks/roles/hosts.update/tasks/centos8.yml

```diff
@@ -0,0 +1,10 @@
+---
+# https://github.com/samba-in-kubernetes/sit-environment/issues/102
+- name: Modify repositories to point CentOS vault
```

Comment:
minor grammar issue: either 'point at CentOS vault' or 'point to CentOS vault'

Reply:
Done.